### ISSUE TITLE
Automate release publish to PyPI

### ISSUE THREAD

**Issue by targos** [nodejs/gyp-next#29]
We could use this GH action: https://github.com/pypa/gh-action-pypi-publish

**Comment by cclauss** [comment]
https://github.com/nodejs/gyp-next/packages is an alternative approach.

**Comment by gengjiawen** [comment]
@targos do you have `PYPI_API_TOKEN` or @ryzokuken has ? This need to configured to settings if I understand correctly.

**Comment by targos** [comment]
I don't have any token.

**Comment by ryzokuken** [comment]
I suppose you'd need a token from my account? Or could I add a new "Node.js" user to the package and we could use the token for that...

**Comment by dennisjackson** [comment]
Is there anything we can do to help this along? PyPI is still stuck on 0.4 and https://github.com/nodejs/gyp-next/pull/83 is now biting new installs.

**Comment by ryzokuken** [comment]
I can post the new release on PyPI manually but we need to figure out something long-term.

**Comment by kpfleming** [comment]
Agreed with @dennisjackson I've just run into this for the first time trying to build an unrelated piece of software which is using gyp-next somewhere in its build system... and my build machine has Python 3.11.

**Comment by ryzokuken** [comment]
Manually updated the package to `0.15.0` on PyPI (https://pypi.org/project/gyp-next/0.15.0/) since it was very outdated and didn't work with Python 3.11 which is what most distros will soon switch to. Please feel free to contact me if you want to set up an automated pipeline for this.

**Comment by cclauss** [comment]
I can work with you on it now that I learned from doing nodejs/tap2junit#49

**Comment by ryzokuken** [comment]
Do we need a Node.js org on PyPI to handle this, or is there a better way?
